include/nbl/ext/CentralLimitBoxBlur/CBlurPerformer.h

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_EXT_BLUR_C_BLUR_PERFORMER_INCLUDED_
+#define _NBL_EXT_BLUR_C_BLUR_PERFORMER_INCLUDED_
+
+#include "nabla.h"
+
+namespace nbl
+{
+namespace ext
+{
+namespace CentralLimitBoxBlur
+{
+
+typedef uint32_t uint;
+struct alignas(16) uvec4
+{
+    uint x, y, z, w;
+};
+#include "nbl/builtin/glsl/ext/CentralLimitBoxBlur/parameters_struct.glsl"
```

Comment:
enclose in `namespace impl`

Reply:
Can we instead make this `private` to `CBlurPeformer`?

Reply:
[Like we did for the GPU blit.](https://github.com/Devsh-Graphics-Programming/Nabla/blob/9b5b2a443608d2fc1fe4da8af85947961cae4e0a/include/nbl/video/utilities/CComputeBlit.h#L9)

Reply:
how would that even work if I wanted to access/muate these members from outside of the class?
I mean the type definition is private, how would that work!? (godbolt example to prove its possible please)

P.S. Does GPU blit have a bug then?